src/Symfony/Component/Cache/Traits/RedisTrait.php

```diff
@@ -271,8 +274,10 @@ public static function createConnection(string $dsn, array $options = [])
         } elseif (is_a($class, \RedisArray::class, true)) {
             foreach ($hosts as $i => $host) {
                 switch ($host['scheme']) {
-                    case 'tcp': $hosts[$i] = $host['host'].':'.$host['port']; break;
-                    case 'tls': $hosts[$i] = 'tls://'.$host['host'].':'.$host['port']; break;
+                    case 'tcp': $hosts[$i] = $host['host'].':'.$host['port'];
+                        break;
+                    case 'tls': $hosts[$i] = 'tls://'.$host['host'].':'.$host['port'];
+                        break;
```

Comment:
let's keep all those on the same line and ignore fabbot on this one

Reply:
Alright, I removed that commit.